amf-client/shared/src/test/resources/cycle/raml10/response/endpoint-emission.yaml

```diff
@@ -7,3 +7,6 @@ get:
           type: object
           examples:
             promotionGetResponse: !include Examples/get-promotion-response.raml
+uriParameters:
```

Comment:
Not sure what caused this change, ideally this should no be present as it is not defined explicitly in the original api definition.